README.md

```diff
@@ -1,10 +1,121 @@
 # carrot-transform
 
-TODO: 
+## Quick Start
```

Comment:
Might be worth just putting this directly into the quickstart here for now: https://github.com/Health-Informatics-UoN/carrot/blob/main/website/src/pages/transform/quickstart.mdx